numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2231,6 +2258,14 @@ def add_newdoc(place, name, doc):
     >>> np.logical_and(x>1, x<4)
     array([False, False,  True,  True, False])
 
+
+    If the inputs are ndarrays, then np.logical_and is equivalent to '&'.
+
+    >>> a = np.array([True,False])
+    >>> b = np.array([False,False])
+    >>> a&b
+    array([False, False])
```

Comment:
This one is incorrect - `&` is `np.bitwise_and`, which happens to behave the same on booleans as `np.logical_and`. You can prove that with
```python
class A(np.ndarray):
    def __array_ufunc__(self, ufunc, method, *args, **kwargs):
        print(ufunc)

np.array(1).view(A) & True
```
which gives
```
<ufunc 'bitwise_and'>
```

It's still worth mentioning `&` here, but noting that it's only appropriate to use on boolean arrays.